static/src/stylesheets/inline/article-labour-liverpool.scss

```diff
@@ -87,6 +92,14 @@ $labour-liverpool-header-element-opacity: .85;
     }
 }
 
+.heroic-immersive{
```

Comment:
Don't need to test here `.heroic-immersive .labour-livepool-headline` is fine

Also, in line with BEM you probably want to describe this a modification of headline like `.headline--labour-liverpool`